mysql-wordpress-pd/README.md

```diff
@@ -4,367 +4,207 @@
+{: .warning}
+
+{% endcapture %}
+
+{% capture objectives %}
+* Create a PersistentVolume
+* Create a Secret
+* Deploy MySQL
+* Deploy WordPress
+* Clean up
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %} 
+
+Download the following configuration files:
+
+1. [local-volumes.yaml](https://kubernetes.io/docs/tutorials/stateful-application/local-volumes.yaml)
+
+1. [mysql-deployment.yaml](https://kubernetes.io//docs/tutorials/stateful-application/mysql-deployment.yaml)
+
+1. [wordpress-deployment.yaml](https://kubernetes.io/docs/tutorials/stateful-application/wordpress-deployment.yaml)
+
+{% endcapture %}
+
+{% capture lessoncontent %} 
+
+## Create a PersistentVolume
+
+MySQL and Wordpress each use a Persistent Volume to store data. While Kubernetes supports many different [types of PersistentVolumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes), this tutorial covers [hostPath](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath).
+
+**Note:** If you have a Kubernetes cluster running on Google Container Engine, please follow [this guide](https://cloud.google.com/container-engine/docs/tutorials/persistent-disk).
+{: .note}
+
+### Setting up a hostPath Volume
+
+A `hostPath` mounts a file or directory from the host node’s filesystem into your Pod. 
+
+**Warning:** Only use `hostPath` for developing and testing. With hostPath, your data lives on the node the Pod is scheduled onto and does not move between nodes. If a Pod dies and gets scheduled to another node in the cluster, the data is lost. 
+{: .warning}
+
+1. Launch a terminal window in the directory you downloaded the manifest files.
+
+2. Create a Persistent Volume from the `local-volumes.yaml` file:
+
+       kubectl create -f local-volumes.yaml
+
+{% include code.html language="yaml" file="local-volumes.yaml" ghlink="/docs/tutorials/stateful-application/local-volumes.yaml %}
+
+{:start="3"} 
+3. Run the following command to verify that two 20GiB PVs are available:
+
+       kubectl get pv
+
+   The response should be like this:
+
+       NAME         CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS      CLAIM     STORAGECLASS   REASON    AGE
+       local-pv-1   20Gi       RWO           Retain          Available                                      1m
+       local-pv-2   20Gi       RWO           Retain          Available                                      1m
+
+## Create a Secret
+
+A [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) is an object that stores a piece of sensitive data like a password or key. The manifest files are already configured to use a Secret, but you have to create your own Secret.
+
+1. Create a file called `password.txt` in the same directory as the manifest files.
+
+2. Set a password by typing your desired password, and then save the file. 
+
+   **Caution:** Make sure that there is no trailing newline at the end of the password.
```

Comment:
if you use --from-literal as I suggested below, you'll get rid of (1), (2) and this "Caution".